resources/models/resource.py

```diff
@@ -187,6 +187,9 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
                                                     null=True, blank=True)
     reservable = models.BooleanField(verbose_name=_('Reservable'), default=False)
     reservation_info = models.TextField(verbose_name=_('Reservation info'), null=True, blank=True)
+    reservation_instructions = models.TextField(
```

Comment:
This should be renamed to `usage_instructions`. `verbose_name` should be just `_('Usage instructions')` and `help_text` should be "Optional instructions regarding the use of the resource, including required guide presence".

Reply:
This change done in 8affc1d.